frontend/src/screens/learner.tsx

```diff
@@ -233,12 +260,13 @@ export const StudiesByLearningPath: FC<{filteredStudies: ParticipantStudy[]}> =
     }
 
     return (
-        <Flex direction='row' w='100%'>
+        <Flex direction='row' w='100%' gap='2.25rem'>
```

Comment:
just FYI For gaps, padding, margin, etc. props on mantine components, they can take values of `xs, sm, md, lg, xl` [here's some more info](https://mantine.dev/styles/style-props/)

Reply:
also this can prob be a <Group> (and is the w=100% needed? worth testing that)

Reply:
Changed it with lg. Removed the w='100%'

Reply:
Changed it with lg. Removed the w='100%'

Reply:
Changed it with lg. Removed the w='100%'